docs/UserGuide.md

```diff
@@ -291,10 +291,10 @@ Format:
 addskill INDEX [s/SKILL]...
 ```
 
-* Only valid indexes are edited. <br>
-  e.g If there are only `4` applicants in the app but `5` is specified, then the intended action will not be carried out.
-* Only positive indexes are edited. <br>
-  e.g As we label our applicants incrementally starting from `1` (e.g `1, 2, 3, and so on`), an index of `-1` will not be tagged to an applicant.
+* Index of applicant specified must not be more than the total number of applicants in LinkedOUT. <br>
+  e.g If there are only `4` applicants in the app but `5` is specified, then the addition will not be carried out as it is invalid.
+* Index of applicant specified must be a positive integer. <br>
```

Comment:
Hmm I think we should probably change the term `integer` to `number` as integer is a more commonly used term in CS, but UG is meant for those without CS background.

Reply:
Made the change